scout/adapter/mongo/variant_events.py

```diff
@@ -559,8 +558,56 @@ def unmark_causative(self, institute, case, user, link, variant):
 
         return updated_case
 
-    def update_manual_rank(self, institute, case, user, link, variant,
-                           manual_rank):
+    def update_cancer_tier(self, institute, case, user, link, variant, cancer_tier):
```

Comment:
What about a test for this one, to increase the coverage?

Reply:
Yep, absolutely!